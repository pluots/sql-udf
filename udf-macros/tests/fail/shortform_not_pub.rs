// Functions must be public

#[udf_macros::udf]
fn foo() {}

fn main() {}
