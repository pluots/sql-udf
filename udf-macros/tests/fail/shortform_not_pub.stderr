error: UDFs must be marked `pub`
 --> tests/fail/shortform_not_pub.rs:3:1
  |
3 | #[udf_macros::udf]
  | ^^^^^^^^^^^^^^^^^^
  |
  = note: this error originates in the attribute macro `udf_macros::udf` (in Nightly builds, run with -Z macro-backtrace for more info)
